integration_tests/src/main/python/delta_lake_write_test.py

```diff
@@ -90,15 +90,16 @@ def test_delta_write_disabled_fallback(spark_tmp_path, disable_conf):
 @delta_lake
 @ignore_order(local=True)
 @pytest.mark.skipif(is_before_spark_320(), reason="Delta Lake writes are not supported before Spark 3.2.x")
-def test_delta_write_round_trip_unmanaged(spark_tmp_path):
+@pytest.mark.parametrize("deletion_vector_conf", deletion_vector_conf_with_reason("https://github.com/NVIDIA/spark-rapids/issues/12027"), ids=idfn)
```

Comment:
Why are we not falling back to the CPU when deletion vectors are enabled for this test instead of just ignoring the deletion vectors? Is it because we do support writing to a table with deletion vectors? just not writing deletes?

Reply:
Ok, it looks like this deals with xfailing the write-cases with deletion-vectors enabled.